lib/Dialect/FIRRTL/Import/FIRParser.cpp

```diff
@@ -1415,6 +1415,30 @@ ParseResult FIRStmtParser::parsePrimExp(Value &result, SubOpVector &subOps) {
     break;                                                                     \
   }
 #include "FIRTokenKinds.def"
+
+  case FIRToken::lp_validif: {
+    auto tloc = translateLocation(loc);
+
+    if (opTypes.size() != 2 || !integers.empty()) {
```

Comment:
Please add a comment here explaining the expansion, and mention this is described in the rationale.

Reply:
Done!